docs/sts-private-bucket.md

```diff
@@ -240,3 +255,113 @@ echo ${OIDC_ARN}
+
+## Steps to Delete<a name="steps-delete"></a>
+
+Those steps should be followed after you've removed the resources created by ccoctl described in [the delete section](./ccoctl.md#aws-delete).
+
+Requirements:
+- You should have set the CloudFront Distribution `Comment` as `${CLUSTER_NAME}`, as described in the section above. Otherwise, you should specify the value you've set when creating the CloudFront Distribution.
+- You should have set the CloudFront Origin Access Identity (OAI) with the field `Comment` with the value `OAI-${OIDC_BUCKET_NAME}`. Otherwise, you should specify the value you've set when creating the CloudFront OAI.
+
+### Remove the Bucket<a name="step-bucket"></a>
+
+If the `ccoctl delete` command failed due to non-empty Bucket (`BucketNotEmpty`), you should follow those steps to complete the Bucket removal.
+
+- Clean the Bucket objects
+
+```bash
+aws s3api delete-object \
+    --bucket ${OIDC_BUCKET_NAME} \
+    --key ".well-known/openid-configuration"
+
+aws s3api delete-object \
+    --bucket ${OIDC_BUCKET_NAME} \
+    --key "keys.json"
+```
+
+- Remove the Bucket using `ccoctl`
+
+```bash
+./ccoctl aws delete \
+    --name=${CLUSTER_NAME} \
+    --region=${CLUSTER_REGION}
+```
+
+### Remove the CloudFront Distribution<a name="step-delete-dist"></a>
+
+CloudFront Distributions can be removed only when it is disabled. To do so, you need to get the current configuration, setting the field `Enabled` to `false`, apply the new configuration, then remove the Distribution.
+
+- Get the CloudFront Distribution ID
+
+```bash
+DISTRIBUTION_ID=$(aws cloudfront list-distributions \
+    --query "DistributionList.Items[?Comment==\`${CLUSTER_NAME}\`].Id" \
+    --output text)
+```
+
+- Get the CloudFront Distribution Config `ETag`
+
+```bash
+ETAG=$(aws cloudfront get-distribution-config \
+    --id ${DISTRIBUTION_ID} \
+    | jq -r '.ETag')
+```
+
+- Get the CloudFront Distribution Configuration, setting the `Enabled` field to `false`
+
+```bash
+aws cloudfront get-distribution-config --id ${DISTRIBUTION_ID} \
+    | jq '.DistributionConfig' \
+    | jq '.Enabled=false' \
+    > ${DIR_CCO}/oidc-cloudfront-to-delete.json
+```
+
+- Apply the new Distribution configuration
+
+```bash
+ETAG=$(aws cloudfront update-distribution \
+    --id ${DISTRIBUTION_ID} \
+    --if-match ${ETAG} \
+    --distribution-config file://${DIR_CCO}/oidc-cloudfront-to-delete.json \
+    | jq -r '.ETag')
+```
+
+- Get the new ETag (it's also returned on the last command).
+
+> The last command updates the `ETAG` variable referencing to a new configuration (Disabled distribution). If you didn't note it you need to re-run the `get-distribution-config` command, as desribed above.
+
+```bash
+aws cloudfront delete-distribution \
+    --id ${DISTRIBUTION_ID} \
+    --if-match ${ETAG}
+```
```

Comment:
```suggestion
- Delete the CloudFront Distribution

```bash
aws cloudfront delete-distribution \
    --id ${DISTRIBUTION_ID} \
    --if-match ${ETAG}
```
```

Reply:
@akhil-rane the suggestion above breaks the code block for markdown (closing the snippet)